Gruntfile.js

```diff
@@ -15,7 +15,7 @@ module.exports = function(grunt) {
 		watch: {
 			sass: {
 				// We watch and compile sass files as normal but don't live reload here
-				files: ['src/assets/**/*.scss'],
+				files: ['src/assets/style.scss'],
```

Comment:
[Change](https://github.com/peterxqe/teamfairtrade/pull/3/commits/b595bf35a9d57691b7cb13f26252d74c0fc178db) is OK.